tests/components/FilterSearch.stories.tsx

```diff
@@ -48,7 +48,7 @@ export default meta;
 export const Primary: StoryFn<FilterSearchProps> = (args) => {
   return (
     <SearchHeadlessContext.Provider value={generateMockedHeadless(mockedHeadlessState)}>
-      <FilterSearch {...args} searchFields={searchFields} />
+      <FilterSearch {...args} searchFields={searchFields} onDropdownInputChange={undefined} />
```

Comment:
nit: can you add this to the `args` object above instead of specifying it inline? it would be similar to how we have "Filter" as the default value for the `label` prop on line 43. that way if we add any stories in the future, we won't have to remember to add this